includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -1919,6 +1920,13 @@ protected function check_response_for_errors( $response ) {
 					$response_code
 				);
 			} elseif ( isset( $response_body['error'] ) ) {
+				if ( isset( $response_body['error']['code'] ) && 'amount_too_large' === $response_body['error']['code'] && 'requires_capture' === $response_body['error']['payment_intent']['status'] ) {
+					throw new Amount_Too_Large_Exception(
+						// translators: This is an error API response.
+						__( 'Error: The payment could not be captured because the requested capture amount is greater than the amount you can capture for this charge.', 'woocommerce-payments' ),
```

Comment:
Just looking into the code, aren't we getting `Error: Error:` in the order note?

Reply:
Good call, but not in this case. I added screenshot above of how the order note looks like with this change.